front/app/api/users/useDeleteSelf.ts

```diff
@@ -35,6 +36,7 @@ const useDeleteSelf = () => {
         invalidateQueryCache();
       }
       clHistory.push('/');
+      scrollToTop();
```

Comment:
I wonder if, instead of calling this util separately, we shouldn't add it as an option to the `clHistory.push` and `clHistory.replace` methods. It would make more sense to have this functionality there as an option, I think.